l1-contracts/contracts/bridge/L1NativeTokenVault.sol

```diff
@@ -46,8 +46,8 @@ contract L1NativeTokenVault is
     /// NOTE: this function may be removed in the future, don't rely on it!
     mapping(uint256 chainId => mapping(address l1Token => uint256 balance)) public chainBalance;
 
-    /// @dev A mapping assetInfo => tokenAddress
-    mapping(bytes32 assetInfo => address tokenAddress) public tokenAddress;
+    /// @dev A mapping assetIdentifier => tokenAddress
+    mapping(bytes32 assetIdentifier => address tokenAddress) public tokenAddress;
```

Comment:
Maybe assetId? We already have chainId, so id instead of identifier makes sense I think